msteams-platform/bots/how-to/format-your-bot-messages.md

```diff
@@ -77,6 +77,19 @@ The following table provides a list of styles, which are supported on desktop, i
 | Hyperlink                 | ✔️ | ✔️ | ✔️ |
 | Image link                | ❌ | ❌ | ❌ |
 
+### AI-based bots
+
+As bots become more conversational, you can leverage AI capabilities to enhance their functionality. Here are some additional features that you can integrate into your bot messages, similar to those used in AI-generated bot messages:
+
+| Feature | Bot | AI bot |
```

Comment:
What does Bot vs AI bot mean here? I thought these are just elements in the message, and any bot can use them.

Reply:
@aosolis, we've updated this section. Could you take a look and let us know if it reads better now?